by-dataframe/dask/README.md

```diff
@@ -0,0 +1,100 @@
+# Connect to CrateDB and CrateDB Cloud using pandas
+
+
+## About
+Example programs demonstrating connectivity with [Dask] and [CrateDB].
+
+This section and examples are mostly about [DataFrame operations with SQLAlchemy],
+specifically about how to insert data into [CrateDB] efficiently.
+
+
+## Usage
+
+The CrateDB Python driver provides a convenience function `insert_bulk`. It
```

Comment:
Does it make sense to add: `insert_bulk` _allows you to efficiently insert multiple rows of data into a CrateDB table in a single operation_?

Reply:
Very good suggestion. I've just amended the patch accordingly. Thanks.